lib/mcb/commands/users/revoke.rb

```diff
@@ -0,0 +1,12 @@
+summary 'Remove a user from an organisation/provider in the DB.'
+param :id_or_email_or_sign_in_id
+param :provider_code, transform: ->(code) { code.upcase }
+usage 'revoke <id or email or sign-in id> <provider_code>'
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+
+  cli = HighLine.new
+  provider = Provider.find_by!(provider_code: args[:provider_code])
+  MCB::RevokeAccessWizard.new(cli, args[:id_or_email_or_sign_in_id], provider).run
```

Comment:
Seeing as we instantiate the `provider` here, I think it would be nice if in keeping with that, it was the concern of this code here to convert the cmdline args into objects. So maybe we should instantiate `user` here (and also validate the arg).

Reply:
Good point. It couldn't be done in grant because grant can optionally create a user. Done.